tests/nightly/nightly-autorun.sh

```diff
@@ -16,9 +16,13 @@
 
 # Get the latest code from vmware/vic repo
 cd ~/go/src/github.com/vmware/vic
-git checkout master
+git checkout -f master
 git remote update
 git rebase upstream/master
+git checkout -f ui-nightlytest/ui-tests \
+    tests/resources/VCH-Util.robot \
+    ui/vic-uia \
+    ui/vic-ui-h5c/uia
```

Comment:
This looks weird. Git merge must have mixed two versions in a wrong way. I'll correct this shortly. FYI, my intent is to only pull in files from the UI test branch and NOT overwrite any existing files.